lighthouse-extension/package.json

```diff
@@ -23,5 +23,8 @@
     "gulp-zip": "^3.2.0",
     "run-sequence": "^1.1.5",
     "through2": "^2.0.1"
+  },
+  "dependencies": {
```

Comment:
if necessary I can publish the github repo under my name in npm repo until the pr is accepted